condor_templates.go

```diff
@@ -118,8 +122,11 @@ const outputTicketListTemplateText = `{{.TicketPathListHeader}}
 
 const interappsSubmissionTemplateText = `universe = vanilla
 executable = /usr/local/bin/interapps-runner
-rank = 100 - TotalLoadAvg{{ if .UsesVolumes }}
-requirements = (HAS_HOST_MOUNTS == True){{ end }}
+rank = 100 - TotalLoadAvg
+requirements = HasDocker && (HAS_CYVERSE_ROAD_RUNNER =?= True){{ if .UsesVolumes }} && (HAS_HOST_MOUNTS =?= True){{ end }}
```

Comment:
Oh, I also added HAS_CYVERSE_ROAD_RUNNER here, because that was intended to be our way of indicating "is set up to run our jobs", not really road-runner specific. It doesn't make a difference for now, because we aren't adding anything that'd make these flock anywhere, but it might at some point.